boa/src/builtins/map/mod.rs

```diff
@@ -2,13 +2,14 @@
 
 use crate::{
     builtins::BuiltIn,
-    object::{ConstructorBuilder, FunctionBuilder, ObjectData, PROTOTYPE},
+    object::{ConstructorBuilder, FunctionBuilder, ObjectData},
     property::{Attribute, DataDescriptor},
     BoaProfiler, Context, Result, Value,
 };
 use ordered_map::OrderedMap;
 
 pub mod map_iterator;
+use crate::object::PROTOTYPE;
```

Comment:
Is there a reason to change the path to `PROTOTYPE`?

Reply:
Just my IDE doing that, when removing and auto-inserting again.